pepper-apis/dsm-core/src/main/resources/liquibase/CMI/PEPPER-903_sent_date.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Needed to get to the date the follow-up survey was created.

Reply:
What uses this?

Reply:
This is the new feature Whitney and Jeff asked for.  They wanted to know when a specific somatic document was sent as a follow-up survey within DSM.